.github/workflows/release.yml

```
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  # create PR if someone manually triggers the event
+  release-please-pr:
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          skip-github-release: true
+  # create a release if we merge a release commit (i.e. merge a PR created above)
+  release-please-release:
+    # match on the default PR title
+    # https://github.com/googleapis/release-please/blob/3569f158158e2bd096a94c145bdb1a7f455829f0/docs/customizing.md#pull-request-title
+    if: "${{ github.event_name == 'push' && contains(github.event.head_commit.message, 'chore(main): release') }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v4
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          skip-github-pull-request: true
```

I think we can keep this file simple for now. I don't expect the same volume of changes as in the uwos-go repo (which is the need for the complexity there: https://github.com/utilitywarehouse/uwos-go/commit/e6b168bd1762125893ca85be577672a2f137bdbe)

```suggestion
name: Release
on:
  push:
    branches:
      - main
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v4
        with:
          token: ${{secrets.GITHUB_TOKEN}}
```

---

[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/41/commits/6ff6fbc4f01c1f74b51d39e1f9b57aee4a8d24eb)